src/main/scala/beam/replanning/AddSupplementaryTrips.scala

```diff
@@ -57,14 +57,17 @@ class AddSupplementaryTrips @Inject()(beamConfig: BeamConfig) extends PlansStrat
         listOfAct.lastOption match {
           case Some(lastAct) =>
             if (lastAct.getType == currentAct.getType) {
-              listOfAct.last.setEndTime(currentAct.getEndTime)
-              listOfAct
+              val lastActivity = PopulationUtils.createActivity(listOfAct.last)
```

Comment:
Should this just be `PopulationUtils.createActivity(lastAct)` or am I missing something

Reply:
this will also work. applied changes as per suggestion